src/ui/UserProfile.h

```diff
@@ -184,7 +184,7 @@ class UserProfile final : public QObject
     Q_INVOKABLE void refreshDevices();
     Q_INVOKABLE void banUser();
     Q_INVOKABLE void signOutDevice(const QString &deviceID);
-    // Q_INVOKABLE void ignoreUser();
+    Q_INVOKABLE void ignoredStatus(const QString &id, const bool ignore);
```

Comment:
Can you make this a Q_PROPERTY? That way you can ignore a user in qml by doing `profile.ignored = true` as well as showing the current status by querying `profile.ignored`.

Reply:
Why would a property not be readily available? You can just read that property on the user object as well as write to it, just like you call the function, but you also get proper change notifications.

Reply:
The difference is that, right now, the profile object the ignoredStatus() function is called on isn't necessarily the user being acted on. It *is* when clicking the ignore button in the profile dialog, but not when unignoring the user from the ignored list (see below). 
https://github.com/Nheko-Reborn/nheko/blob/7c9fed91b82c174c40aefdce009090dd142b76f9/resources/qml/dialogs/IgnoredUsers.qml#L65 
I suppose a possible branch of that question would be whether I could easily get a profile object from the user id.